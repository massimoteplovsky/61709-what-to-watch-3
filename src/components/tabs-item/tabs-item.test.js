import React from "react";
import renderer from "react-test-renderer";
import TabsItem from "./tabs-item.jsx";

it(`<TabsItem /> component renders correctly`, () => {
  const tree = renderer
    .create(<TabsItem
      id={0}
      onChangeActiveItemIndex={() => {}}
      title={``}
      activeItemIndex={0}
    />);

  expect(tree).toMatchSnapshot();
});
