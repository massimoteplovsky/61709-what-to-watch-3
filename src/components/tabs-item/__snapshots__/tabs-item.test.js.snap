// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabsItem /> component renders correctly 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="#"
    onClick={[Function]}
  >
    
  </a>
</li>
`;
